### PR TITLE
Feat: Set up i18n

### PR DESCRIPTION
**Pull request by Andy-Golden**
## Task
https://trello.com/c/fCkdsxlm/44-set-up-i18n-and-apply-storybook

## Description
- Set up i18n
- Apply storybook

## Screenshot

https://github.com/Andy-Golden/find-place-to-live/assets/147120986/fafb6fed-ad95-4e16-8818-cd687799947d

### CODE REVIEW

File: public/locales/vi/translation.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "vi": "vi",
+  "email": "Địa chỉ email",
+  "password": "Mật khẩu",
+  "enterEmail": "Nhập địa chỉ email",
+  "enterPassword": "Nhâp mật khẩu",
+  "signUp": "Đăng ký",
+  "signIn": "Đăng nhập",
+  "signUpPage": {
+    "subTitle": "Tạo tài khoản để sử dụng nhiều tính năng",
```
Comment (carl-huynh-goldenowl) [inline public/locales/vi/translation.json]: This translation does not match with _Create your account to get full access_

File: src/common/SelectLanguage/helper.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import type { RootState } from "@store";
+import { switchLang } from "store/slices";
+
+import { Lang } from "@enums";
+
+import type { ISelectPrepareHook } from "./interfaces";
+
+const useSelectLanguagePrepareHook = (): ISelectPrepareHook => {
+  const { i18n } = useTranslation();
+
```
Comment (carl-huynh-goldenowl) [inline src/common/SelectLanguage/helper.ts]: Remove this line.

File: src/common/SelectLanguage/helper.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const lang = useSelector((state: RootState) => state.langReducer.lang);
+
```
Comment (carl-huynh-goldenowl) [inline src/common/SelectLanguage/helper.ts]: ditto

**Review by carl-huynh-goldenowl — CHANGES_REQUESTED** [review]

**Comment by carl-huynh-goldenowl** [comment]
Missing space
Screen Shot 2023-11-15 at 00 12 42

**Comment by Andy-Golden** [comment]
I changed font to Jost san-serif
image

**Review by carl-huynh-goldenowl — APPROVED** [review]
LGTM

**Pull request merged**